### PR TITLE
doc(lcm): Add mbpo doc

### PR DESCRIPTION
**Pull request by Will-Nie**
chuming adds mbpo algo benchmark doc

### CODE REVIEW

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/hands_on/mbpo.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+MBPO
+^^^^^^^
+
+Overview
+---------
+Model-based policy optimization (MBPO) was first proposed in `When to Trust Your Model: Model-Based Policy Optimization <https://arxiv.org/abs/1906.08253>`_. MBPO utilizes short model-generated rollouts and provides a guarantee of monotonic improvement at each step. In details, MBPO trains an ensemble of models to fit the transitions of the true environment and uses it to generate short trajectories starting from real environment states to do policy improvement. For the choice of RL policy, MBPO use SAC as its RL component.
```
Comment (jiaruonan) [inline source/hands_on/mbpo.rst:6]: ** trains an ensemble of [probabilistic] models **

File: source/hands_on/mbpo.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+4. MBPOuses **model-eensemble**.
+
+
+Key Equations or Key Graphs
+---------------------------
+The maximum likelihood loss used in model training is:
+
+.. math::
+
+   L(w)=\mathbb{E}\left[log( f^w(\boldsymbol{s}_{t+1}|\boldsymbol{s}_t,\boldsymbol{a}_t))\right]
```
Comment (jiaruonan) [inline source/hands_on/mbpo.rst:25]: 建议模型符号f和参数w改为p和θ，与下面伪代码中的模型表示相对应。

File: source/hands_on/mbpo.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Pseudo-code
+---------------
+.. image:: images/MBPO.png
+  :align: center
+  :scale: 55%
+
+.. note::
+  The initial implementation of MBPO only give the hyper-parameters of applying it on SAC, which does not fit DDPG and TD3 well.
+
```
Comment (jiaruonan) [inline source/hands_on/mbpo.rst:35]: 补充一些related work: Extensions ?

File: source/hands_on/mbpo.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Quick Facts
+-------------
+1. MBPO is a **model-based** RL algorithm.
+
+2. MBPO uses SAC as its RL policy.
+
+3. MBPO only support **continuous** action spaces.
+
+4. MBPOuses **model-eensemble**.
```
Comment (jiaruonan) [inline source/hands_on/mbpo.rst:16]: 补充一点：5. The model usage of MBPO is to do k-length rollouts from replay buffer.

File: source/hands_on/mbpo.rst (hunk 3 above)
Comment (lichuming) [reply source/hands_on/mbpo.rst:35]: Trust the Model When It Is Confident:
Masked Model-based Actor-Critic
https://proceedings.neurips.cc/paper/2020/file/77133be2e96a577bd4794928976d2ae2-Paper.pdf
加上这篇作为extension

**Pull request merged**